src/quo/components/wallet/account_permissions/view.cljs

```diff
@@ -0,0 +1,94 @@
+(ns quo.components.wallet.account-permissions.view
+  (:require [quo.components.avatars.account-avatar.view :as account-avatar]
+            [quo.components.dividers.divider-line.view :as divider-line]
+            [quo.components.icon :as icons]
+            [quo.components.markdown.text :as text]
+            [quo.components.selectors.selectors.view :as selectors]
+            [quo.components.wallet.account-permissions.style :as style]
+            [quo.components.wallet.address-text.view :as address-text]
+            [quo.components.wallet.required-tokens.view :as required-tokens]
+            [quo.foundations.colors :as colors]
+            [quo.theme :as quo.theme]
+            [react-native.core :as rn]
+            [schema.core :as schema]
+            [utils.i18n :as i18n]))
+
+(def ?schema
+  [:=>
+   [:catn
+    [:props
+     [:map {:closed true}
+      [:account
+       [:map {:closed true}
+        [:name [:maybe :string]]
+        [:address [:maybe :string]]
+        [:emoji [:maybe :string]]
+        [:customization-color [:maybe [:or :string :keyword]]]]]
+      [:token-details {:optional true} [:maybe [:vector required-tokens/?schema]]]
+      [:keycard? {:optional true} [:maybe :boolean]]
+      [:checked? {:optional true} [:maybe :boolean]]
+      [:disabled? {:optional true} [:maybe :boolean]]
+      [:on-change {:optional true} [:maybe fn?]]
+      [:theme :schema.common/theme]]]]
+   :any])
+
+(defn view-internal
+  [{:keys
+    [checked? disabled? on-change token-details keycard? theme]
+    {:keys
+     [name address emoji customization-color]} :account}]
```

Comment:
Maybe due to linter, but IMO is easier to read if the vector of keys is next to the keyword instead of below:
`{:keys [a b c d]}` instead of 

```
{:keys 
  [a b c d]}
```

Reply:
<img width="774" alt="Screenshot 2023-12-20 at 10 33 05" src="https://github.com/status-im/status-mobile/assets/19279756/5a0986dd-1fba-461e-a6d2-97701f0be7b2">

If I keep the `:keys` keywords and vector on the same line, the linter pushes the vector too much, and it looks weird to me. That's why I opted to keep them on two separate lines.

Reply:
I prefer the current version, what do you guys think should I move it to a single line?

Reply:
@ajayesivan, I think either way is fine, we're fighting the formatter in the end.

This is how I would format this code if zprint could understand that a blank line between destructuring sections shouldn't be aligned. I learned this trick from @yqrashawn.

```clojure
(defn- view-internal
  [{:keys [checked? disabled? on-change token-details keycard? theme]

    {:keys [name address emoji customization-color]} :account}]
  [rn/view
   ...])
```